frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -61,6 +61,30 @@ class AcsPopulationProvider extends VariableProvider {
         population_pct: (row) => percent(row.population, totalStatePopulation),
       });
     });
+
+    // Race data comes from backend with Total, if it's not wanted, we remove it.
+    if (
+      !breakdowns.includeTotal &&
+      (breakdowns.demographic === "race_nonstandard" ||
+        breakdowns.demographic === "race")
+    ) {
+      df = df.where((row) => row["race_and_ethnicity"] !== "Total");
+    }
+
+    // Age and sex data comes from backend without Total, if it's wanted, we add it.
```

Comment:
I think it might be better to move this above line 51, so that you don't have to special-case the population_pct logic anymore. Also makes it easier to drop when the server gives us Totals. Ie something like this:

```
if (breakdowns.age || breakdowns.sex) {
  // Add "Total" for those
}

df = applyToGroups((group) => {
  // Calculate population_pct, using "Total" regardless of breakdown
});

if (!breakdowns.includeTotal) {
  df = df.where(...) // filter out "Total"
}
```